### ISSUE TITLE
Can't reproduce the result for BurstSR dataset(Track2)

### ISSUE THREAD

**Issue by Egkang-Luis** [akshaydudhane16/Burstormer#5]
Hi, thanks for your wonderful work. I am also working on BurstSR and your codes have helped me a lot.

While I have some problems when I try to fine-tune an SR model on the BurstSR dataset. I firstly train the SR model on the Synthetic dataset and everything is OK. Then I fine-tune this model on the BurstSR dataset. The training loss keeps decreasing while the validation PSNR only grows at the beginning and drops gradually after it achieves the best result. It seems that the training set is overfitted.

Have you met the same problem? I would appreciate it a lot if you could help me figure this out.

Thanks.

**Comment by Egkang-Luis** [comment]
I'm waiting your response :)

**Comment by cvkaiming** [comment]
> I'm waiting your response :)

Hi, did you encounter any errors while running the Track 2 code, I found that when loading samsung_00\im_raw.png, it reported a path error, can you please tell me what you did in the Burst Super-resolution\data_sets\burstsr_dataset.py ? Thank you, bro.

**Comment by Egkang-Luis** [comment]
> > I'm waiting your response :)
> 
> Hi, did you encounter any errors while running the Track 2 code, I found that when loading samsung_00\im_raw.png, it reported a path error, can you please tell me what you did in the Burst Super-resolution\data_sets\burstsr_dataset.py ? Thank you, bro.

Could you share your including your error comment?

**Comment by cvkaiming** [comment]
> > > I'm waiting your response :)
> > 
> > 
> > Hi, did you encounter any errors while running the Track 2 code, I found that when loading samsung_00\im_raw.png, it reported a path error, can you please tell me what you did in the Burst Super-resolution\data_sets\burstsr_dataset.py ? Thank you, bro.
> 
> Could you share your including your error comment?

When I run Burstormer_Track_2_training.py directly I get the error in the image, after debugging I found that at SamsungRAWImage.load('{}/{}/samsung_{:02d}'.format(self.root, self.burst _list[burst_id], im_id)) This line of code is reporting an error, the path does not include the name of the subfolder under the train dataset train00 folder for example. I assume you were able to run it successfully, so that should have solved the problem
![image](https://github.com/akshaydudhane16/Burstormer/assets/109211594/014bef21-b512-4113-b820-af644a93a866)
thanks for your reply

**Comment by Egkang-Luis** [comment]
> > > > I'm waiting your response :)
> > > 
> > > 
> > > Hi, did you encounter any errors while running the Track 2 code, I found that when loading samsung_00\im_raw.png, it reported a path error, can you please tell me what you did in the Burst Super-resolution\data_sets\burstsr_dataset.py ? Thank you, bro.
> > 
> > 
> > Could you share your including your error comment?
> 
> When I run Burstormer_Track_2_training.py directly I get the error in the image, after debugging I found that at SamsungRAWImage.load('{}/{}/samsung_{:02d}'.format(self.root, self.burst _list[burst_id], im_id)) This line of code is reporting an error, the path does not include the name of the subfolder under the train dataset train00 folder for example. I assume you were able to run it successfully, so that should have solved the problem ![image](https://user-images.githubusercontent.com/109211594/258842282-014bef21-b512-4113-b820-af644a93a866.png) thanks for your reply

Honestly, I don't remember clearly. But I didn't have any error that position as I remember.

Could you check your dataset folder?

**Comment by cvkaiming** [comment]
When testing at SyntheticBurst, I get strange images，can you help me？
![0005](https://github.com/akshaydudhane16/Burstormer/assets/109211594/d26e3568-2b1c-4c59-b6fe-6501d413d8a0)
